### PR TITLE
set up Github actions

### PR DESCRIPTION
**Pull request by d-morrison**
added:

- spell check
- tidyverse style check ("lint")

### CODE REVIEW

File: data-raw/papers.R
Comment (d-morrison) [inline data-raw/papers.R:1]: fixed style issues

File: data-raw/papers2.R
Comment (d-morrison) [inline data-raw/papers2.R:1]: deleted instead of fixing style issues

File: .github/workflows/lint-project.yaml
Comment (d-morrison) [inline .github/workflows/lint-project.yaml:1]: checks R files for adherence to tidyverse style; reports any issues

File: .github/workflows/check-spelling.yaml
Comment (d-morrison) [inline .github/workflows/check-spelling.yaml:1]: (checks spelling)

File: R/create_pub_listing.R
Comment (d-morrison) [inline R/create_pub_listing.R:1]: edits for style

File: R/read_bib_file.R
Comment (d-morrison) [inline R/read_bib_file.R:1]: edits for style

File: index.qmd
Comment (d-morrison) [inline index.qmd:1]: `underrecognized` was flagged by the spell-checker

File: publications/articles/aiemjoy2022/aiemjoy2022lancet.qmd
Comment (d-morrison) [inline publications/articles/aiemjoy2022/aiemjoy2022lancet.qmd:1]: removed unused publications/ directory to avoid lints

File: R/read_bib_file.R
Comment (d-morrison) [inline R/read_bib_file.R:1]: replaced with `bibtex::read.bib()`

**Review by kaiemjoy — APPROVED** [review]

**Pull request merged**